inference-engine/tests/functional/plugin/shared/include/single_layer_tests/activation.hpp

```diff
@@ -70,10 +70,11 @@ static std::map<ngraph::helpers::ActivationTypes, std::string> activationNames =
         {ngraph::helpers::ActivationTypes::Mish,        "Mish"},
         {ngraph::helpers::ActivationTypes::HSwish,      "HSwish"},
         {ngraph::helpers::ActivationTypes::SoftPlus,    "SoftPlus"},
+        {ngraph::helpers::ActivationTypes::Swish,       "Swish"},
 };
 
 typedef std::tuple<
-        ngraph::helpers::ActivationTypes,
+        std::pair<ngraph::helpers::ActivationTypes, std::vector<float>>,
```

Comment:
Please, add comment. It's hard to understand why `std::pair<ngraph::helpers::ActivationTypes, std::vector<float>>`